pom.xml

```diff
@@ -48,7 +48,8 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <skipTests>false</skipTests>
         <skipUnitTests>${skipTests}</skipUnitTests>
-        <skipIntegrationTests>${skipTests}</skipIntegrationTests>
+        <!--skipIntegrationTests>${skipTests}</skipIntegrationTests-->
+        <skipIntegrationTests>true</skipIntegrationTests>
```

Comment:
skipping all integration test?

Reply:
yes, all IT TCs are being skipped as they need SF account details including private keys etc.